packages/client/src/__tests__/strictEnum.test.ts

```diff
@@ -0,0 +1,35 @@
+import { makeStrictEnum } from '../runtime/strictEnum'
```

Comment:
What do you think about keeping the spec/unit tests near their parent file instead of a separate folder?
VSCode has a neat feature to collapse those into a single entry in the file browser panel.
```
    "explorer.fileNesting.patterns": {
        "*.js": "${capture}.js.map, ${capture}.min.js, ${capture}.d.ts",
        "*.jsx": "${capture}.js",
        "*.tsx": "${capture}.ts",
        "tsconfig.json": "tsconfig.*.json",
        "package.json": "package-lock.json, yarn.lock",
        "*.ts": "${capture}.spec.ts" // <= collapsing spec files
    },
```

Reply:
Nice, will do just that.